### PR TITLE
Fix section guide

### PR DESCRIPTION
**Pull request by maulanayuseph**
**Related Task**
https://app.clickup.com/t/2gp3nnq

**Preview**
screencapture-localhost-8083-isoman-2022-04-05-13_54_48

**Evidence**
title: Fix section guide
project: Pikobar Website
participants: @maulanayuseph @Ibwedagama @bangunbagustapa @naufalihsank

### CODE REVIEW

File: components/_pages/index/IsolasiMandiri/index.vue [hunk 1]
```diff
@@ -16,6 +16,7 @@
       <ExpandableContent
         v-for="(item, i) in infoItems"
         :key="i"
+        :open="i === 0 ? true : false"
```
Comment (naufalihsank) [inline components/_pages/index/IsolasiMandiri/index.vue:19]: hello, maybe `:open="i === 0"` would be enough ?

**Review by naufalihsank — APPROVED** [review]

**Review by bangunbagustapa — APPROVED** [review]

File: components/_pages/index/IsolasiMandiri/index.vue [hunk 2]
```diff
@@ -26,7 +27,10 @@
         />
       </ExpandableContent>
     </template>
-    <div class="isoman__action-card-grids">
+    <div
+      v-show="false"
+      class="isoman__action-card-grids"
+    >
```
Comment (Ibwedagama) [inline components/_pages/index/IsolasiMandiri/index.vue:33]: ask:

hello @maulanayuseph is this element hidden on purpose?

Comment (maulanayuseph) [reply components/_pages/index/IsolasiMandiri/index.vue:33]: yes @Ibwedagama this element is hidden and maybe this element will appear in the future.

File: components/_pages/index/IsolasiMandiri/index.vue (hunk 1 above)
Comment (maulanayuseph) [reply components/_pages/index/IsolasiMandiri/index.vue]: hi thanks fixed on this [commit](https://github.com/jabardigitalservice/pikobar-jabarprov-go-id/pull/492/commits/4d8cc619046ea72f4b0883dd24cb0e5029df20ad)

**Review by Ibwedagama — APPROVED** [review]

**Pull request merged**